bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py

```diff
@@ -124,7 +124,10 @@ def _map_data(row):
 def _get_name(row):
     first_name = row.get("first_name", "")
     middle_name = row.get("middle_name", "")
-    return f"{first_name} {middle_name}".strip()
+    name = f"{first_name} {middle_name}".strip()
+    if row.get("organization_name") and name == "None":
```

Comment:
Do I get it right that based on the fix, it looks like the name "None" is an actual string from OATS DB?

Reply:
That's correct, which is why we were seeing the first name field be populated with "None"

Reply:
the organization_name check is probably unnecessary but technically someone's first name could be None